website/integrations/services/immich/index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Immich
+---
+
+<span class="badge badge--secondary">Support level: Community</span>
+
+## What is Immich
+
+> Immich is a self-hosted backup solution for photos and videos on mobile devices
+>
+> -- https://immich.app/
+
+## Preparation
+
+The following placeholders will be used:
+
+-   `https://immich.company` is the URL used to access the Immich instance.
+-   `authentik.company` is the FQDN of the authentik install.
+
+## authentik Configuration
+
+1. Create a new OAuth2/OpenID Provider
```

Comment:
```suggestion
1. Create a new OAuth2/OpenID Provider using the following settings:
```

Reply:
Fixed in [07febfa](https://github.com/goauthentik/authentik/pull/7829/commits/07febfaaac6b7c4dade7d39178006215ebf728e3)